website/docs/tutorials/07-testing/02-cheatcodes/expect-events.md

```diff
@@ -1,28 +1,15 @@
 # `expect_events`
 ```python
- def expect_events(
-            *raw_expected_events: Union[
-                str, # Protostar interprets string as an event's name 
-                TypedDict("ExpectedEvent", {
-                    "name": str,
-                    "data": NotRequired[Union[
-                      List[int],
-                      Dict[
-                        # e.g.
-                        # {"current_balance" : 37, "amount" : 21}
-                        # 
-                        # for the following event signature:
-                        # @event
-                        # func balance_increased(current_balance : felt, amount : felt):
-                        # end
-                        DataTransformer.ArgumentName,
-                        DataTransformer.SupportedType,
-                      ]
-                    ]],
-                    "from_address": NotRequired[int]
-                },
-            )],
-        ) -> None: ...
+class EventExpectationDict(TypedDict):
+    name: str
+    data: NotRequired[list[int] | dict[str, Any]]
+    from_address: NotRequired[int]
+
+EventExpectationName = str
+
+EventExpectation = EventExpectationDict | EventExpectationName
```

Comment:
I would call it `ExpectedEvent` imo fits better. Expectation suggests behaviour not data

Reply:
Done